Content.Server.Database/Entity/PostgresServerDbContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Content.Server.Database.Entity
+{
+    using System;
+    using System.ComponentModel.DataAnnotations.Schema;
+    using System.Net;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Storage;
+
+    public sealed class PostgresServerDbContext : ServerDbContext
+    {
+        public PostgresServerDbContext()
+        {
+        }
+        public PostgresServerDbContext(DbContextOptions<ServerDbContext> options) : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder options)
+        {
+            //DO NOT COMMIT
```

Comment:
:thinking:

Reply:
Whoops, that seems to have slipped. Accidentally committed when figuring out the empty constructors that apparently are required for migration and without which it sets some default options thus breaking `options.IsConfigured`.